addons/website_event/views/event_templates.xml

```diff
@@ -11,710 +11,796 @@
-        <div class="modal-dialog modal-lg">
+    <div id="modal_attendees_registration" class="modal fade" tabindex="-1" role="dialog">
+        <div class="modal-dialog modal-lg" role="document">
             <form id="attendee_registration" t-attf-action="/event/#{slug(event)}/registration/confirm" method="post" class="js_website_submit_form">
                 <input type="hidden" name="csrf_token" t-att-value="request.csrf_token()"/>
                 <div class="modal-content">
-                    <header class="modal-header">
-                        <h4 class="modal-title" id="myModalLabel"><strong>Attendees</strong></h4>
-                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">x</button>
-                    </header>
-                    <main class="modal-body">
-                        <div class="container">
-                            <t t-set="counter_type" t-value="1"/>
-                            <t t-set="counter" t-value="0"/>
-                            <t t-foreach="tickets" t-as="ticket">
-                                <h4 class="o_page_header">
-                                    <strong>
-                                        Ticket Type: <t t-esc="ticket['name']"/>
-                                        <t t-if="ticket['price'] == 0">(Free)</t>
-                                    </strong>
-                                </h4>
-                                <div class="row d-none d-lg-flex">
-                                    <div class="col-md">
-                                        <strong>Name</strong>
+                    <div class="modal-header align-items-center">
+                        <h4 class="modal-title">Attendees</h4>
+                        <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span>&amp;times;</span></button>
+                    </div>
+                    <t t-set="counter_type" t-value="1"/>
+                    <t t-set="counter" t-value="0"/>
+                    <t t-foreach="tickets" t-as="ticket">
+                        <t t-foreach="range(1, ticket['quantity'] + 1)" t-as="att_counter" name="attendee_loop">
+                            <t t-set="counter" t-value="counter + 1"/>
+                            <div class="modal-body bg-light border-bottom">
+                                <h5 class="mt-1 pb-2 border-bottom">Ticket #<span t-esc="counter"/> <small class="text-muted">- <span t-esc="ticket['name']"/></small></h5>
+                                <div class="row">
+                                    <div class="col-lg my-2">
+                                        <label>Name</label>
+                                        <input class="form-control" type="text" t-attf-name="#{counter}-name" required="This field is required"/>
                                     </div>
-                                    <div clas
```

Comment:
Mmh, not sure about this, complexify the code for nothing according to me (not even sure how to handle that with our python routes system. I think that the user should just add a link to the event page to its 404 default page if he wants to... I think we can think about that for 14.0, you don't use it anywhere yet right ? I'd remove it for 13.0

Reply:
Github bug... I am talking about this here: https://github.com/odoo/odoo/pull/37009/files/c350c79abffa58306f01dbf909a43e33630752b8#diff-10bb56317eeadceee415973bccef7455R738

Reply:
Yes. Not even sure if it's a good idea / really useful to have this.